src/app/static/src/js/types.ts

```diff
@@ -0,0 +1,19 @@
+export interface Error {
+    response?: {
+        data?: {
+            errors?: any[]
+        }
+    }
+}
+
+export interface ReportDependency {
+    id: string,
+    name: string,
+    out_of_date: boolean,
```

Comment:
What's the intended use of this? (not saying there isn't one... just isn't clear!)

Reply:
Well, we're not using it at the moment though it does come back from orderly server. We may add an out of date indicator to the UI in future. It could come out of the front end time until then.

Reply:
I was not aware of this, but if report B is `out_of_date` it simply means that B depends on A and A has been been run more recently than B.